File: spring-boot-java/src/main/java/io/ssaragi/java/spring/boot/dto/response/exam/ExamDataResModel.java
package io.ssaragi.java.spring.boot.dto.response.exam;

import io.ssaragi.java.spring.boot.dto.data.exam.ExamData;
import io.ssaragi.java.spring.boot.dto.response.BaseApiResponseModel;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@EqualsAndHashCode(callSuper = false)
public class ExamDataResModel extends BaseApiResponseModel {

    private ExamData data;
}

File: spring-boot-kotlin/src/main/kotlin/io/ssaragi/kotlin/spring/boot/dto/response/exam/ExamDataResModel.kt
package io.ssaragi.kotlin.spring.boot.dto.response.exam

import io.ssaragi.kotlin.spring.boot.dto.data.exam.ExamData
import io.ssaragi.kotlin.spring.boot.dto.response.BaseApiResponseModel

data class ExamDataResModel(

    var data: ExamData? = null,

) : BaseApiResponseModel()
